elftools/construct/lib/binary.py

```diff
@@ -15,13 +15,7 @@ def int_to_bin(number, width=32):
     """
     if number < 0:
         number += 1 << width
-    i = width - 1
```

Comment:
Why is this change required?

Reply:
Not required, but adding type hints requires reading and understanding the code. My brain has the (bad) habit to rewrite the code to be more pregnant. Moved to #609